stable/kong/README.md

```diff
@@ -410,3 +410,15 @@ You can can learn about kong ingress custom resource definitions [here](https://
 
 - DB-less mode is enabled by default.
 - Kong is installed as an Ingress Controller for the cluster by default.
+
```

Comment:
Please add a changelog entry for 0.27.1 itself. A note on how it updates only the changelog is all we need.